modules/layout/css/stories.tsx

```diff
@@ -5,9 +5,9 @@ import README from './README.md';
 import './index.scss';
 import './stories.scss';
 
-storiesOf('CSS/Grid System', module)
+storiesOf('CSS/Layout', module)
   .addDecorator(withReadme(README))
-  .add('Responsive Grids', () => (
+  .add('Responsive Layout', () => (
     <div className="story story-grid">
```

Comment:
@sahlhoff btw, is `story-grid` needed? I don't see it referenced anywhere else, but I wanted to double-check with you before removing it.

Reply:
@jamesfan good to remove 👍